app-modules/student-data-model/src/Filament/Resources/StudentResource.php

```diff
@@ -123,21 +123,21 @@ public static function getPages(): array
     {
         return [
             'index' => ListStudents::route('/'),
-            'manage-alerts' => ManageStudentAlerts::route('/{record}/alerts'),
-            'manage-engagement' => ManageStudentEngagement::route('/{record}/engagement'),
-            'manage-files' => ManageStudentFiles::route('/{record}/files'),
-            'manage-form-submissions' => ManageStudentFormSubmissions::route('/{record}/form-submissions'),
-            'manage-application-submissions' => ManageStudentApplicationSubmissions::route('/{record}/application-submissions'),
-            'manage-information' => ManageStudentInformation::route('/{record}/information'),
-            'manage-interactions' => ManageStudentInteractions::route('/{record}/interactions'),
-            'manage-subscriptions' => ManageStudentSubscriptions::route('/{record}/subscriptions'),
-            'manage-tasks' => ManageStudentTasks::route('/{record}/tasks'),
+            // 'manage-alerts' => ManageStudentAlerts::route('/{record}/alerts'),
+            // 'manage-engagement' => ManageStudentEngagement::route('/{record}/engagement'),
+            // 'manage-files' => ManageStudentFiles::route('/{record}/files'),
+            // 'manage-form-submissions' => ManageStudentFormSubmissions::route('/{record}/form-submissions'),
+            // 'manage-application-submissions' => ManageStudentApplicationSubmissions::route('/{record}/application-submissions'),
+            // 'manage-information' => ManageStudentInformation::route('/{record}/information'),
+            // 'manage-interactions' => ManageStudentInteractions::route('/{record}/interactions'),
+            // 'manage-subscriptions' => ManageStudentSubscriptions::route('/{record}/subscriptions'),
+            // 'manage-tasks' => ManageStudentTasks::route('/{record}/tasks'),
             'view' => ViewStudent::route('/{record}'),
-            'timeline' => StudentEngagementTimeline::route('/{record}/timeline'),
-            'care-team' => ManageStudentCareTeam::route('/{record}/care-team'),
-            'service-management' => StudentServiceManagement::route('/{record}/service-management'),
-            'events' => ManageStudentEvents::route('/{record}/events'),
-            'programs' => ManageStudentPrograms::route('/{record}/programs'),
+            // 'timeline' => StudentEngagementTimeline::route('/{record}/timeline'),
+            // 'care-team' => ManageStudentCareTeam::route('/{record}/care-team'),
+            // 'service-management' => StudentServiceManagement::route('/{record}/service-management'),
+            // 'events' => ManageStudentEvents::route('/{record}/events'),
+            // 'programs' => ManageStudentPrograms::route('/{record}/programs'),
```

Comment:
You will want to do a search for `StudentResource::getUrl` and fix any references to these old pages to now just go directly to the Student View page.